indra/tests/test_pybel_assembler.py

```diff
@@ -115,14 +112,15 @@ def test_modification_with_evidences():
     edge_data = get_edge_data(belgraph, braf_dsl, map2k1_mod_dsl)
     assert edge_data.get(pc.SUBJECT) == activity('kin')
     assert edge_data[pc.RELATION] == pc.INCREASES
-    assert edge_data[pc.EVIDENCE] == 'evidence text'
```

Comment:
this should still always be available, so I would leave it with `[]` instead of `.get()`

Reply:
It's just to make the test FAIL instead of ERROR if the key is missing (which it was  before I updated the test), it's fine either way though.

Reply:
ah okay